### ISSUE TITLE
IDs not replaced for namespaced SVGs

### ISSUE THREAD

**Issue by waruyama** [iconfu/svg-inject#22]
SVGs can be namespaced, which means that tag names can have a namespace prefix like `<svg:style>`. Currently, IDs are not replaced if elements have a namespace prefix.

Here is a simple example.

index.xhtml:
```xhtml
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:svg="http://www.w3.org/2000/svg">
  <head>
    <script src="svg-inject.js"></script>
  </head>
  <body>
    <img src="svg1.svg" onload="SVGInject(this)"/>
  </body>
</html>
```

svg1.svg:
```xml
<svg:svg xmlns:svg="http://www.w3.org/2000/svg" width="300" height="200">
  <svg:style>
    .cls2 {
      fill: url(#grad2);
    }
  </svg:style>
  <svg:defs>
    <svg:linearGradient id="grad2">
      <svg:stop offset="0" stop-color="red" />
      <svg:stop offset="1" stop-color="green" />
    </svg:linearGradient>
  </svg:defs>
  <svg:circle cx="150" cy="100" r="50" class="cls2"/>
</svg:svg>
```

Although the SVG is displayed correctly, the IDs are not replaced, which can result in the known problems when multiple SVGs are injected.

**Comment by waruyama** [comment]
If an element is namespaced, we can use the localName property instead of the tagName property. Getting the tag name without the namespace prefix would then look as simple as this:

`var tagName = elem.localName || elem.tagName;`

**Comment by waruyama** [comment]
Apparently `localName` is supported down to Internet Explorer 9. Therefore we can simply replace `elem.tagName` with `elem.localName`.

**Comment by iconfu** [comment]
just released Version 1.2.0 which fixes this issue
